vite-project/src/components/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { IcArrow, IcHome } from '../assets/svgs/icons';
+
+const Header = () => {
+    return (
+        <Head>
+            <IcArrow />
+            <HeadTextWrapper>
+                장바구니 <HeadTextBox>(2)</HeadTextBox>
+            </HeadTextWrapper>
+            <IcHome />
```

Comment:
여기서 Icon들 같은 경우에는 어쨌거나 버튼의 역할 또는 페이지 이동의 역할을 하는거잖아? 그러면 결국 기능 구현을 해야한다고 생각했을 때 각각을 알맞는 button 또는 Link로 감싸 주어야겠지!
```suggestion
            <Button>
                <IcArrow />
            </Button>
            <HeadTextWrapper>
                장바구니 <HeadTextBox>(2)</HeadTextBox>
            </HeadTextWrapper>
            <Link>
                <IcHome />
            </Link>
```

여기서 내가 말한 `Link` 태그는 react-router의 Link태그야! 그래서 지금은 react-router를 안 깔았으니까, 나중에 react-router 라이브러리가 있을 때 사용하면 좋을 것 같아!

지금은 두 개 아이콘 다 button 태그로만 감싸주는 정도로만 연습하면 좋을 것 같네요! ㅎㅎ

button 태그는 스타일 컴포넌트로 만들어서 감싸면 됩니다!

Reply:
여기서 button이 스타일 컴포넌트인데도 감싸줘야하는거야??
아니면 Link가 이후에 기능 구현했을 때 있으면 차이를 주기 위해서 한다고 생각하면 되나??

Reply:
아니아니 저 아이콘은 지금 사실 뜯어보면 <svg ~~ 인거니까 역할에 따라서 button 태그나 Link 알맞는걸로 감싸주면 된다는 얘기였어!

그 과정에서
const Button = styled.button

이렇게 만들어서 가져오면 되겠다 ~~ 한거였어 스타일링 필요할거라서 ㅎㅎ